packages/contracts/test/contracts/OVM/chain/OVM_CanonicalTransactionChain.gas.spec.ts

```diff
@@ -55,7 +56,7 @@ describe('[GAS BENCHMARK] OVM_CanonicalTransactionChain', () => {
     )
     await AddressManager.setAddress(
       'OVM_DecompressionPrecompileAddress',
-      DECOMPRESSION_ADDRESS
+      predeploys.Lib_AddressManager
```

Comment:
Hmmmmm ok I need to take a closer look at this when I have a chance. Seems like we mixed up a few addresses in our tests but doesn't seem like the addresses used actually have an impact on the tests. But probably good to fix the mistake now to avoid more confusion later. Will get back to this Monday.

Reply:
Ok taking a look at this now. I believe we have a typo in this test. Should be `predeploys.OVM_SequencerEntrypoint` instead of `predeploys.Lib_AddressManager`.